File: windows/Model/AGCountry.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AppGo.Model
{
    [Serializable]
    public class AGCountry
    {
        [JsonProperty("id")]
        public int id { get; set; } = 0;

        [JsonProperty("name")]
        public string name { get; set; } = "";

        [JsonProperty("alias_zh")]
        public string alias_zh { get; set; } = "";

        [JsonProperty("alias_en")]
        public string alias_en { get; set; } = "";

        [JsonProperty("description")]
        public string description { get; set; } = "";
    }
}

File: android/mobile/src/main/java/com/appgo/appgopro/models/AGCountry.kt
package com.appgo.appgopro.models

import com.google.gson.annotations.SerializedName

/**
 * Created by KSMA on 4/19/2017.
 */

class AGCountry {

    @SerializedName("id")
    var id: Int = 0

    @SerializedName("name")
    var name: String? = null

    @SerializedName("alias_zh")
    var aliasZh: String? = null

    @SerializedName("alias_en")
    var aliasEn: String? = null

    @SerializedName("description")
    var description: String? = null
}
